troxy-embedded/pom.xml

```diff
@@ -15,6 +15,10 @@
             <groupId>no.sparebank1.troxy</groupId>
             <artifactId>troxy-core</artifactId>
         </dependency>
+        <dependency>
+            <groupId>no.sparebank1.troxy</groupId>
+            <artifactId>troxy-server</artifactId>
+        </dependency>
```

Comment:
Tror att tanken var att `troxy-server` ska innehålla det som trengs før att køra troxy standa-lone, medan `troxy-embedded` ska innehålla det krævs før att køra troxy i embedded modus... då blir detta lite feil.

Reply:
Du har kanske lagt in avhængigheten før att funktionaliteten ska kunna testas med integrationstester som benytter troxy-embedded? Vad med att flytta ut statestikhanteringen i  `troxy-core`?  Då kan statestik REST-endepunkterna ligga kvar i `troxy-server` och nær man brukar `troxy-embedded` kan man efterfråga statestik vha. java API:t.

Reply:
jepp, var egentligen bara før att kunna skriva test ja. Men enig, ska se på det